hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java

```diff
@@ -154,6 +160,10 @@ public class TestSCMNodeManager {
   private static final LayoutVersionProto CORRECT_LAYOUT_PROTO =
       toLayoutVersionProto(MAX_LV, MAX_LV);
 
+//  public TestSCMNodeManager() {
+//    this.clock = Clock.system(ZoneId.systemDefault());
```

Comment:
It would be good to create as instance variable instead of in each test method.  But should use `TestClock`, not `Clock.system()`.